src/android/PendingUpload.java

```diff
@@ -0,0 +1,47 @@
+package com.spoon.backgroundfileupload;
+
+import android.util.Log;
+
+import com.orm.SugarRecord;
+import com.orm.dsl.Unique;
+import com.orm.query.Condition;
+import com.orm.query.Select;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.util.List;
+
+public class PendingUpload extends SugarRecord {
+    String uploadId;
+    String data;
+
+    public PendingUpload() {}
+
+    public PendingUpload(JSONObject payload) {
+        try {
+            uploadId = payload.getString("id");
+            data = payload.toString();
+        } catch (JSONException e) {
+            Log.d("CordovaBackgroundUpload", "eventLabel:'error reading id during PendingUpload creation'");
```

Comment:
Logs are parsed with key=value :smile: 
```suggestion
            Log.d("CordovaBackgroundUpload", "eventLabel='error reading id during PendingUpload creation'");
```